lib/common/ServiceInfo.ts

```diff
@@ -0,0 +1,46 @@
+import ServiceVersionModel from './models/ServiceVersionModel'; 
+
+/**
+ * Encapsulates the functionality to get the information about the service such as
+ * version info.
+ */
+export default class ServiceInfo {
+
+    private static readonly packageJson = require('../../package.json');

```

Comment:
We use [standardjs](https://standardjs.com/rules.html) style which uses 2 spaces for indents.